include/IECorePython/Wrapper.h

```diff
@@ -1,96 +0,0 @@
-//////////////////////////////////////////////////////////////////////////
```

Comment:
As that'll require a rebase to fix, shall I also rebase in the ParameterWrapper stuff discussed above, or would you prefer that as separate new commits?